apps/judicial-system/web/src/components/AccordionItems/IndictmentsCaseFilesAccordionItem/IndictmentsCaseFilesAccordionItem.tsx

```diff
@@ -195,55 +237,107 @@ const CaseFile: React.FC<CaseFileProps> = (props) => {
           <Text>{caseFile.displayText.split('|')[1]}</Text>
         </Box>
       ) : (
-        <Box
-          display="flex"
-          justifyContent="spaceBetween"
-          alignItems="center"
-          background="blue100"
-          padding={2}
+        <div
+          className={styles.caseFileWrapper}
           onPointerUp={() => {
+            if (isDragging) {
+              onReorder(caseFile.id)
+            }
             setIsDragging(false)
-            onReorder(caseFile.id)
           }}
         >
-          <Box display="flex" alignItems="center">
-            <Box
-              data-testid="caseFileDragHandle"
-              display="flex"
-              marginRight={3}
-              style={{ cursor: isDragging ? 'grabbing' : 'grab' }}
-              onPointerDown={(e) => {
-                setIsDragging(true)
-                controls.start(e)
-              }}
-            >
-              <Icon icon="menu" color="blue400" />
-            </Box>
-            <Box
-              display="flex"
-              alignItems="center"
-              component="button"
-              onClick={() => {
-                if (caseFile.id) {
-                  onOpen(caseFile.id)
-                }
-              }}
-            >
-              <Text variant="h5">{caseFile.displayText}</Text>
-              <Box marginLeft={1}>
-                <Icon icon="open" type="outline" size="small" />
-              </Box>
-            </Box>
+          <Box
+            data-testid="caseFileDragHandle"
+            display="flex"
+            paddingX={3}
+            paddingY={2}
+            style={{ cursor: isDragging ? 'grabbing' : 'grab' }}
+            onPointerDown={(e) => {
+              setIsDragging(true)
+              controls.start(e)
+            }}
+          >
+            <Icon icon="menu" color="blue400" />
           </Box>
-          <Box display="flex" alignItems="center">
-            <Box display="flex" marginRight={3}>
-              <Text variant="small">{formatDate(caseFile.created, 'P')}</Text>
-            </Box>
-            <button onClick={() => alert('not implemented')}>
-              <Icon icon="pencil" color="blue400" />
-            </button>
+          <Box width="full">
+            <AnimatePresence initial={false} exitBeforeEnter>
+              {editFileId === caseFile.id ? (
+                <motion.div
+                  initial={{ y: 10, opacity: 0 }}
+                  animate={{ y: 0, opacity: 1 }}
+                  exit={{ y: 10, opacity: 0 }}
+                  transition={{ duration: 0.5 }}
```

Comment:
Consider making this 200ms. Short animations should be under 200ms. Here is a good ref: https://m3.material.io/styles/motion/easing-and-duration/tokens-specs